org.yacas.yacas.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id": "org.yacas.yacas",
+    "branch": "1.6",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.4",
```

Comment:
Not at all, it just was originally made when 1.4 was the current version  and I must have missed the updates. Thanks!